lib/acme_server/jws.ex

```diff
@@ -25,8 +25,7 @@ defmodule AcmeServer.JWS do
       [{_jwk, [{true, payload, _jws}]}] ->
         {:ok, %{payload: Jason.decode!(payload), protected: protected}}
 
-      _ ->
-        :error
+        error -> error
```

Comment:
This violates the typespec in line 2, which is possibly the cause of the dialyzer warnings.